frontend/src/store/modules/auth.js

```diff
@@ -34,6 +38,7 @@ export default {
         localStorage.setItem('token', token);
         localStorage.setItem('refresh_token', refresh_token);
         axios.defaults.headers.common['Authorization'] = 'Bearer ' + token;
+        commit('setToken', token).then(router.push('/dashboard'));
```

Comment:
Not sure what it is about, but the problem seems to be traced to here.

![Zrzut ekranu z 2020-07-29 22-32-19](https://user-images.githubusercontent.com/28675024/88850288-6fb6eb80-d1eb-11ea-9c02-28b4f2dec1aa.png)